src/RelExpr.js

```diff
@@ -151,18 +160,18 @@ class RelExpr extends Component<Props> {
    */
   handleExprClick(e: SyntheticMouseEvent<HTMLElement>) {
     e.stopPropagation();
-    if (this.props.changeExpr) {
-      const target = e.target instanceof HTMLElement ? e.target : undefined;
-      const parent =
-        target && target.parentElement instanceof HTMLElement
-          ? target.parentElement
-          : undefined;
-      this.props.changeExpr(this.props.expr, parent);
-      this.props.ReactGA.event({
-        category: 'User Selecting Relational Algebra Enclosure',
-        action: Object.keys(this.props.expr)[0],
-      });
-    }
+    const node =
+      ReactDOM.findDOMNode(this) instanceof HTMLElement
+        ? ReactDOM.findDOMNode(this)
+        : undefined;
+
+    if (node instanceof HTMLElement && this.props.changeExpr)
+      this.props.changeExpr(this.props.expr, node);
```

Comment:
Always use braces around if statements to avoid potential problems in the future.